.cypress/integration/7_app_analytics.spec.js

```diff
@@ -307,6 +303,7 @@ describe('Viewing application', () => {
   it('Opens span detail flyout when Span ID is clicked', () => {
     cy.get('[data-test-subj="app-analytics-traceTab"]').click();
     cy.wait(delay);
+    cy.get('input[type="search"]').focus().type(`5ff3516909562c60`);
```

Comment:
Search for specific traceId to prevent flakiness on it not showing up on first page